client/tsconfig.json

```
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "outDir": "./dist/",
+    "noImplicitAny": true,
+    "module": "es6",
+    "target": "es5",
+    "jsx": "react",
+    "allowJs": true
```

Can you also add DOM/ES libs?
```suggestion
    "allowJs": true,
    "lib": [
      "dom",
      "dom.iterable",
      "es6"
    ],
```